<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(2, 83, 157)</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>100</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="lbIDschet">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>IDschet</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbIDkv">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>IDkv</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDate">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSumSchet">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>SumSchet</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDatePay">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
    <property name="text">
     <string>DatePay</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="leIDschet">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIdkv">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDate">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSumSchet">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStatus">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDatePay">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 164, 127)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOpen">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInsert">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDelete">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOplata">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Оплачено</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNOplata">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Не оплачено</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 164, 127)</string>
   </property>
   <property name="text">
    <string>Поиск по ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
